low-level/solution/lr2/server/manifest.py

```diff
@@ -67,7 +67,7 @@ def __init__(self):
             'system': 'hw>system[%s]>%s',
```

Comment:
Class mapping of manifest and health are different. 
How can we relate manifest and health resource paths?

Reply:
Some of the data we were not able to map because both manifest and health source is different for server data.
Currently, we are trying to map with possible resources only.
like we are targeting mainly disk, PSU, and fan. and remaining all SW services or whatever we can map.

Reply:
>How can we relate manifest and health resource paths?

@mariyappanp for this exactly we align rpath till resource type in both health & manifest, like
server>hw>disk

and now power->psu being changed here at line 70 for same reason i believe

Reply:
Okay